README.md

```diff
@@ -19,7 +19,7 @@ import { updateRepo } from "@artsy/update-repo"
 await updateRepo({
   repo: { owner: "artsy", repo: "metaphysics" },
   branch: "update-npm-dependency",
-  targetBranch: "master",
+  targetBranch: "main",
```

Comment:
I'm hoping that https://github.com/artsy/update-repo/pull/52 fixes the missing behavior around `targetBranch`.